crates/holochain/examples/websocket_stress.rs

```diff
@@ -0,0 +1,125 @@
+use holochain::{
+    conductor::{
+        api::{AdminRequest, AdminResponse},
+        Conductor,
+    },
+};
+use tempfile::TempDir;
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_conductor_api::conductor::KeystoreConfig;
+use holochain_conductor_api::AdminInterfaceConfig;
+use holochain_conductor_api::InterfaceDriver;
+use tokio_tungstenite::*;
+use tungstenite::Message;
+use futures::sink::SinkExt;
+use futures::stream::StreamExt;
+use holochain_websocket::WireMessage;
+
+use std::sync::atomic::{AtomicU64, Ordering};
+
+static ID: AtomicU64 = AtomicU64::new(1);
+
+use holochain_serialized_bytes as hsb;
+
+const MINUTES_LONG_BEHAVED_COUNT: usize = 10;
+const MINUTES_LONG_BAD_COUNT: usize = 10;
+const SECONDS_LONG_BEHAVED_COUNT: usize = 100;
+const SECONDS_LONG_BAD_COUNT: usize = 100;
+
+static CONS_MADE: AtomicU64 = AtomicU64::new(0);
+static MSGS: AtomicU64 = AtomicU64::new(0);
+static GOOD_CLOSE: AtomicU64 = AtomicU64::new(0);
+static BAD_CLOSE: AtomicU64 = AtomicU64::new(0);
```

Comment:
This does look like metrics to me. I would like to see these embedded in Holochain under the `otel` feature flag rather than measured from here. I've deliberately held off re-targeting the metrics work from `develop-0.1` to `develop` and preparing for merge because I don't want to be putting anything big/risky into `develop` like new dependencies and feature flags while we're trying to focus on bug fixes but would that be useful to anybody if we put some time into it?

Reply:
I'm certainly up for updating these into some standardized form once we've come up with that standardization. But, in terms of bang-for-buck, this here is an experiment that currently must be manually run to understand the results. I don't think reporting the metrics anywhere provides any benefit.